packages/icons/icons.json

```
@@ -3562,6 +3562,13 @@
         "group": "miscellaneous",
         "codepoint": 61728
     },
+    {
+        "displayName": "Object Array",
+        "iconName": "array-object",
+        "tags": "data, type, list, object",
+        "group": "miscellaneous",
+        "codepoint": 61729
```

Can we add this entry to the bottom of the icon manifest?
```suggestion
        "codepoint": 62356
```